src/data/cakeLpPools.json

```diff
@@ -37,6 +37,25 @@
       "decimals": "1e18"
     }
   },
+  {
+    "name": "cakev2-ampl-bsc-mp-busd",
+    "address": "0x6e98beb694ff1cdb1ee130edd2b21b0298683d58",
```

Comment:
Can you checksum `address` please? 

0x6E98Beb694Ff1CDb1eE130eDD2b21b0298683d58